src/index.html

```diff
@@ -7,12 +8,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
```

Comment:
Fix class everywhere
```suggestion
      class="card"
```

Reply:
<img width="442" alt="Снимок экрана 2024-06-12 в 12 08 22" src="https://github.com/mate-academy/layout_product-cards/assets/169082749/fd462a3b-a799-4c16-80c6-b23b3e5e893a">
 у мене цього немає